### ISSUE TITLE
Add temporary permisions to whitelist

### ISSUE THREAD

**Issue by msxfm** [RequestPolicyContinued/requestpolicy#294]
<a href="https://github.com/Ghoughpteighbteau"><img src="https://avatars.githubusercontent.com/u/1427632?" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [Ghoughpteighbteau](https://github.com/Ghoughpteighbteau)**
_Sunday Feb 26, 2012 at 00:31 GMT_
_Originally opened as https://github.com/RequestPolicy/requestpolicy/issues/294_

---

A workflow improvement. A typical user story might go like this:
![How to requestpolicy](http://i.imgur.com/ZpT7p.png)

The problem is that last bit (not living the long happy life part, that's a problem too, though...). What's a quick intuitive way to confirm a temporary permission? 

I think it's here: 
![Where to put it](http://i.imgur.com/9eGhc.png)

Right under the sub menu of a temporarily authorized destination. 
"Permanently allow destination" Would be a nice option.

**Comment by Cerberus-tm** [comment]
This workflow (the second one) is exactly what I do. I have learned to no longer use temporary permissions at all, because at the moment making a single temporary permission permanent involved opening the menu twice and reloading the entire page twice, which takes too much time. (I usually only need to make one permanent: the rest should normally be revoked.) So I strongly support a right-click option to make a single temporary permission permanent.

**Comment by nodiscc** [comment]
Maybe a click on a rule's timer icon (requested in #167, mockup in https://github.com/RequestPolicyContinued/requestpolicy/issues/601#issue-57730116) could make a particular temporary rule permanent? If so, ther should be a tooltip indicating what clicking on the icon does.

**Comment by Cerberus-tm** [comment]
That sounds great! As long as no page-reloading is necessary and it can be done with one or two clicks.

**Comment by nodiscc** [comment]
> Maybe the timer icon should look like a button, then

The hand icon on hover should be sufficient

**Comment by Cerberus-tm** [comment]
I agree, a hand icon will be sufficient once you learn that it is there (although I think a tooltip would be nice for new users who otherwise might not discover the hidden function).
